src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  static addToAlive(animal) {
+    this.alive.push(animal);
+  }
+
+  static removeFromAlive(animal) {
+    if (this.alive.includes(animal)) {
+      const indexOfAnimal = this.alive.indexOf(animal);
+
```

Comment:
interesting idea. but a bit hard;)

Reply:
yep, I thought that we have to do this by static methods :))